tests/mol-tts_multi/EPUB/package.opf

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Media Overlays</dc:coverage>
+    <dc:creator>Ivan Herman</dc:creator>
+    <dc:date>2022-02-17</dc:date>
+    <dc:description>Tests whether the Reading System implements Text-to-Speech as a fallback when no audio content is provided. Severak fragments are defined in the SMIL file.</dc:description>
+    <dc:identifier id="pub-id">mol-tts_multi</dc:identifier>
+    <dc:language>en</dc:language>
+    <dc:title>Multiple text-to-speech fallback for MO</dc:title>
+    <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
+    <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
+    <meta property="belongs-to-collection">should</meta>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#sec-text-to-speech</meta>
```

Comment:
Since there are two separate normative statements in this section, I think they should have distinct anchors and you should point to the one under test (or both individually).

Reply:
Right. I have not yet handled the anchors; doing it this time.